app/src/main/java/com/mmm/parq/activities/HostActivity.java

```diff
@@ -21,6 +22,8 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onClick(View view) {
                 Snackbar.make(view, "This will add a new spot!", Snackbar.LENGTH_LONG)
                         .setAction("Action", null).show();
+                Intent i = new Intent(getApplicationContext(), HostNewSpotActivity.class);
```

Comment:
What is the difference between using getApplicationContext() and just passing in 'this'?